### PR TITLE
✨ Add team section

### PR DESCRIPTION
**Pull request by feliciofilipe**
Added a team section to the landing. Note that the footer is in the
wrong color - this fix is pending on #24.

All the team information comes from a team.json file on the data/ folder.
I updated the Header to do the proper routing to this page.

### CODE REVIEW

File: data/team.json [hunk 1]
```diff
@@ -0,0 +1,147 @@
...
+    },
+    {
+        "picture": "brandao.jpg",
+        "name": "Miguel Brandão",
+        "role": "Developer"
+    },
+    {
+        "picture": "lelis.jpg",
+        "name": "Vitor Lelis",
+        "role": "Diretor de Curriculo e Material"
```
Comment (nelsonmestevao) [inline data/team.json:25]:
```suggestion
        "role": "Responsável do Currículo e Material"
```

File: landing/Header/index.js [hunk 2]
```diff
@@ -11,16 +11,22 @@ function Header() {
...
+              alt="CoderDojo Braga Logo"
+            />
+          </a>
         </div>
         <ul className="hidden md:flex items-center gap-12 text-black text-sm uppercase">
           <li className="cursor-pointer hover:text-primary">Currículo</li>
           <li className="cursor-pointer hover:text-primary">Projetos</li>
-          <li className="cursor-pointer hover:text-primary">Equipa</li>
+          <li className="cursor-pointer hover:text-primary">
+            <a href="team">
```
Comment (nelsonmestevao) [inline landing/Header/index.js:26]: Use Link component from `next/link` to navigate between pages.

File: landing/Header/index.js [hunk 3]
```diff
@@ -53,7 +59,11 @@ function Header() {
             <ul className="flex flex-col items-center text-xl gap-6 uppercase">
               <li className="cursor-pointer hover:text-primary">Currículo</li>
               <li className="cursor-pointer hover:text-primary">Projetos</li>
-              <li className="cursor-pointer hover:text-primary">Equipa</li>
+              <li className="cursor-pointer hover:text-primary">
+                <a href="./team.js">
```
Comment (nelsonmestevao) [inline landing/Header/index.js:63]:
```suggestion
                <a href="./team">
```

File: pages/team.js [hunk 4]
```diff
@@ -0,0 +1,54 @@
+import { Header, Member, Footer } from "@landing";
+
+import team from "data/team.json";
```
Comment (nelsonmestevao) [inline pages/team.js:3]:
```suggestion
import team from "~data/team.json";
```

File: landing/Header/index.js [hunk 5]
```diff
@@ -53,7 +58,9 @@ function Header() {
             <ul className="flex flex-col items-center text-xl gap-6 uppercase">
               <li className="cursor-pointer hover:text-primary">Currículo</li>
               <li className="cursor-pointer hover:text-primary">Projetos</li>
-              <li className="cursor-pointer hover:text-primary">Equipa</li>
+              <li className="cursor-pointer hover:text-primary">
+                <a href="./team">Equipa</a>
```
Comment (nelsonmestevao) [inline landing/Header/index.js:62]:
```suggestion
                <Link href="/team">Equipa</Link>
```

**Review by nelsonmestevao — CHANGES_REQUESTED** [review]
Missing things:

- [x] The content should align with the limits of navbar and footer;
- [x] The breakpoints should be improved, for example, on mobile should be one person per row, and the description should also decrease its font size as the title does;
- [x] We need a description;
- [ ] The "Saber mais" button should not have transparent background around and the text on hover shouldn't be blue.

File: styles/Home.module.css [hunk 6]
```diff
@@ -557,7 +557,7 @@
     width: 100%;
     height: 28%;
     background-color: white;
-    transform: translate(0%, 1435px);
+    transform: translate(0%, 1435.5px);
```
Comment (nelsonmestevao) [inline styles/Home.module.css:560]:
```suggestion
    transform: translate(0%, 1435px);
```

**Review by ruioliveira02 — APPROVED** [review]

**Pull request merged**